tests/test_adjoint_jacobian.py

```diff
@@ -384,27 +369,6 @@ class TestAdjointJacobianQNode:
     def dev_kokkos(self):
         return qml.device("lightning.kokkos", wires=2)
 
-    def test_finite_shots_warning(self):
-        """Tests that a warning is raised when computing the adjoint diff on a device with finite shots"""
-
-        dev = qml.device("lightning.kokkos", wires=1, shots=1)
-
-        with pytest.warns(
-            UserWarning,
-            match="Requested adjoint differentiation to be computed with finite shots.",
-        ):
-
-            @qml.qnode(dev, diff_method="adjoint")
-            def circ(x):
-                qml.RX(x, wires=0)
-                return qml.expval(qml.PauliZ(0))
-
-        with pytest.raises(
-            NotImplementedError,
-            match="lightning.kokkos does not currently support finite shots",
-        ):
-            qml.grad(circ)(0.1)
-
```

Comment:
Since this is still true (adjoint doesn't support finite shots) we should keep this test and adjust the adjoint code to ensure we get the warning. I assume this still exists in the `lightning.qubit` tests ?

Reply:
Thanks, you're right! Let me fix it